packages/plugin-ext/src/plugin/plugin-manager.ts

```diff
@@ -30,7 +30,7 @@ import {
 } from '../common/plugin-api-rpc';
 import { PluginMetadata, PluginJsonValidationContribution } from '../common/plugin-protocol';
 import * as theia from '@theia/plugin';
-import { join } from 'path';
+import { join } from './path';
```

Comment:
There is a file `env.ts` in this directory, but no `path.ts`

Reply:
`path.ts` is a new file with necessary `join` and `normalize` implementations (like in `path` package).
We need then to be able to run the script on the browser side.